pages/challenge.tsx

```diff
@@ -0,0 +1,32 @@
+import Head from "next/head"
+
+export default function Web() {
+  return (
+    <>
+      <Head>
+        <meta property="og:url" content="https://next-enterprise.vercel.app/" />
+        <meta
+          property="og:image"
+          content="https://raw.githubusercontent.com/Blazity/next-enterprise/main/project-logo.png"
```

Comment:
Can we please remove these meta-tags? These doesnt make sens for us

Reply:
done